yarn-project/end-to-end/src/fixtures/utils.ts

```diff
@@ -115,6 +124,28 @@ const getACVMConfig = async (logger: DebugLogger) => {
   }
 };
 
+// Determines if we have access to the bb binary and a tmp folder for temp files
+export const getBBConfig = async (logger: DebugLogger) => {
```

Comment:
PR #6051 adds the same function. Just leaving a note here to make sure one of the implementations is cleaned up once both PRs land.